networking/multiple_networks/understanding-multiple-networks.adoc

```diff
@@ -57,4 +57,6 @@ networks in your cluster:
 
  * *macvlan*: xref:../../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-macvlan-object_configuring-additional-network[Configure a macvlan-based additional network] to allow pods on a host to communicate with other hosts and pods on those hosts by using a physical network interface. Each pod that is attached to a macvlan-based additional network is provided a unique MAC address.
 
+  * *tap*: xref:../../networking/multiple_networks/configuring-additional-network.adoc#nw-multus-tap-object_configuring-additional-network[Configure a tap-based additional network] to create a tap device inside the container namespace. Tap devices in containers are virtual network interfaces used to connect a container to a virtual network. They are used to provide a bridged network connection between the container and the host operating system, allowing the container to communicate with other devices on the network.
```

Comment:
where did you get this? PLease read this https://www.kernel.org/doc/html/v5.8/networking/tuntap.html

Reply:
I had that in my section about the tap plugin referenced from https://www.cni.dev/plugins/current/main/tap/ will revise based on your recommendation

Reply:
that link doesn't mention anything of what's written there

Reply:
Hi @mlguerrero12 I made some changes trying to simplify, let me know what you think.